test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpath-plugin.yaml

```diff
@@ -225,6 +225,8 @@ spec:
             - "--v=5"
             - "--endpoint=$(CSI_ENDPOINT)"
             - "--nodeid=$(KUBE_NODE_NAME)"
+            - "--enable-controller-modify-volume=true"
```

Comment:
This file is synced from an external project via the update-hostpath.sh script. so it's not recommended to modify this file in k/k.

You can change the container configure in `PrepareTest` before deploying it. 

There's another PR doing similar behavior. 
- https://github.com/kubernetes/kubernetes/pull/123151